velox/connectors/hive/HiveDataSink.h

```diff
@@ -200,6 +200,7 @@ class FileNameGenerator : public ISerializable {
 
   virtual std::pair<std::string, std::string> gen(
       std::optional<uint32_t> bucketId,
+      std::optional<uint32_t> maxBucketCount,
```

Comment:
Is the `maxBucketCount` used to generate the file name?

Reply:
@majetideepak Yes. https://github.com/facebookincubator/velox/blob/main/velox/connectors/hive/HiveDataSink.cpp#L167-L175